src/css_parser.rs

```diff
@@ -2948,6 +3041,11 @@ mod css_tests {
         assert_eq!(parse_pixel_value("aslkfdjasdflk"), Err(PixelParseError::InvalidComponent("aslkfdjasdflk")));
     }
 
+    #[test]
+    fn test_parse_pixel_value_5() {
+        assert_eq!(parse_pixel_value("0"), Ok(PixelValue::px(0.)));
```

Comment:
This will not parse for anything besides "0" (ex.`"1"` won't parse), and also I'd just leave "px is implicit" out, as I mentioned above.